### PR TITLE
Add Azure Tenant ID as environment variable

### PR DESCRIPTION
**Pull request by EmandM**

### CODE REVIEW

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 1]
```diff
@@ -1,20 +1,21 @@
 apiVersion: v1
 appVersion: 1.0
-description: Helm chart for deploying Azure Policy add-on in Azure Arc Kubernetes Connected Clusters
-name: azure-policy-addon-arc-connected-clusters
-version: 0.2.0
+description: Helm chart for deploying Azure Policy add-on in Kubernetes
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: Helm chart for deploying Azure Policy add-on in Azure Arc Kubernetes clusters.

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+name: azure-connected-clusters-policy-addon
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: Can you change it to azure-policy-addon-arc-connected-clusters?

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+version: 0.1.0
 keywords:
   - azure
   - azure policy
   - azure-policy
   - governance
   - compliance
-  - arc connected clusters
-  - arc-connected-clusters
+  - acs connected clusters
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: arc connected clusters

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+  - acs-connected-clusters
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: arc-connected-clusters

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 5]
```diff
@@ -1,20 +1,21 @@
 apiVersion: v1
 appVersion: 1.0
-description: Helm chart for deploying Azure Policy add-on in Azure Arc Kubernetes Connected Clusters
-name: azure-policy-addon-arc-connected-clusters
-version: 0.2.0
+description: Helm chart for deploying Azure Policy add-on in Kubernetes
+name: azure-connected-clusters-policy-addon
+version: 0.1.0
 keywords:
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml:6]: Can you add below keywords?
gatekeeper
opa
Kubernetes policy

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 6]
```diff
@@ -1,20 +1,21 @@
...
   - azure-policy
   - governance
   - compliance
-  - arc connected clusters
-  - arc-connected-clusters
+  - acs connected clusters
+  - acs-connected-clusters
   - kubernetes
 home: https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks?toc=/azure/aks/toc.json
+# icon: TODO:: Add icon
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: Remove the icon line

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 7]
```diff
@@ -1,20 +1,21 @@
...
 sources:
   - https://github.com/Azure/azure-policy-kubernetes
 maintainers:
-  - name: AzureDG
-    email: azuredg@microsoft.com
\ No newline at end of file
+  - name: emmcmill
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: AzureDG

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 8]
```diff
@@ -1,20 +1,21 @@
...
+    email: emmcmill@microsoft.com
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: azuredg@microsoft.com

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/deployment.yaml [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
+{{- if (and (ne .Values.azurepolicy.env.resourceid "<your_resource_id>") (and (ne .Values.azurepolicy.env.tenantid "<your_tenant_id>") (and (ne .Values.azurepolicy.env.clientid "<your_client_id>") (ne .Values.azurepolicy.env.clientsecret "<your_client_secret>")))) }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: azure-policy
   namespace: kube-system
 type: Opaque
 data:
-  client-secret: {{ .Values.azurepolicy.env.clientsecret | b64enc }}
+  client-secret: {{ .Values.azurepolicy.env.clientsecret | b64enc | quote }}
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/deployment.yaml:9]: Is quote needed? Can we remove it?

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/ns-label.yaml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/ns-label.yaml:1]: Can you also add label to azure-arc namespace?

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/sync.yaml [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+        version: "v1"
+        kind: "Namespace"
+      - group: ""
+        version: "v1"
+        kind: "Pod"
+      - group: ""
+        version: "v1"
+        kind: "Service"
+      - group: "extensions"
+        version: "v1beta1"
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/sync.yaml:10]: Can you just have ingress in this file as other types are not required for sync? I think ingress is the only referential policy we have. Please check.

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/values.yaml [hunk 12]
```diff
@@ -5,8 +5,8 @@
 ## Microsoft Azure Policy Add-on in Kubernetes
 azurepolicy:
   image:
-    name: "emmcmill/azure-policy-kubernetes"
-    tag: "ccs-test"
+    name: "mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod"
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/values.yaml]: Is this the latest image to use for tomorrow's bug bash? For now, you can specify test image.

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/values.yaml [hunk 13]
```diff
@@ -5,8 +5,8 @@
...
+    tag: "prod_20191011.1"
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/values.yaml]: Same with the tag as well.

**Comment by RamyasreeChakka** [comment]
@EmandM Did you try installing and uninstalling using this helm chart? Did you see any issues? Did install and delete happen cleanly?

**Review by RamyasreeChakka — CHANGES_REQUESTED** [review]

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/deployment.yaml (hunk 9 above)
Comment (EmandM) [reply Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/deployment.yaml:9]: It's needed. Ensures the secret gets deployed with quotation marks around it which ensures no ambiguous characters get picked up

File: Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml [hunk 14]
```diff
@@ -1,20 +1,26 @@
...
   - azure-policy
   - governance
   - compliance
+  - gatekeeper
+  - opa
+  - kubernetes policy
+  - kubernetes-policy
   - arc connected clusters
   - arc-connected-clusters
+  - acs connected clusters
```
Comment (RamyasreeChakka) [inline Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/Chart.yaml]: Remove acs connected clusters

**Review by RamyasreeChakka — APPROVED** [review]

**Pull request merged**